### PR TITLE
Handles some decryption error + allow to decrypt without Step 1

### PR DESCRIPTION
**Pull request by vzotova**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -84,9 +84,9 @@ export default function App() {
   };
 
   const decryptMessage = async (encryptedMessageId: string) => {
-    if (!condition) {
-      return;
-    }
+    // if (!condition) {
+    //   return;
+    // }
```
Comment (derekpierre) [inline src/App.tsx]: Let's just remove these lines.

File: src/App.tsx [hunk 2]
```diff
@@ -96,15 +96,19 @@ export default function App() {
...
 
-    const decryptedMessage = await decrypt(
-      provider,
-      domain,
-      encryptedMessage,
-      'https://porter-tapir.nucypher.io/',
-      provider.getSigner(),
-    );
+    try {
+      const decryptedMessage = await decrypt(
```
Comment (derekpierre) [inline src/App.tsx:107]: This demo is using an older version of the API - https://github.com/nucypher/taco-ardrive-demo/blob/master/package.json#L17. Let's update to the latest version, `@nucypher/taco` `0.5.0`, and update the call to `decrypt` appropriately. You can use the code from here as an example - https://github.com/nucypher/taco-web/blob/main/examples/taco/nextjs/src/hooks/useTaco.ts#L36.

**Review by derekpierre — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 2 above)
Comment (vzotova) [reply src/App.tsx:107]: ✔️

File: src/App.tsx (hunk 1 above)
Comment (vzotova) [reply src/App.tsx]: ✔️

**Review by derekpierre — APPROVED** [review]
🎸

**Pull request merged**